crates/egui/src/widgets/plot/mod.rs

```diff
@@ -1081,6 +1081,21 @@ impl PlotUi {
         self.bounds_modified = true.into();
     }
 
+    /// Scale the plot bounds around a position in screen coordinates. Can be useful for
+    /// implementing alternative plot navigation methods.
+    pub fn zoom_bounds(&mut self, zoom_factor: Vec2, center: Pos2) {
+        self.last_screen_transform.zoom(zoom_factor, center);
+        self.bounds_modified = true.into();
+    }
+
+    /// Scale the plot bounds around the hovered position, if any. Can be useful for implementing
+    /// alternative plot navigation methods.
+    pub fn zoom_bounds_around_hovered(&mut self, zoom_factor: Vec2) {
```

Comment:
These needs to document what the `zoom_factor` means. i.e. how do I zoom in/make things bigger vs how do I zoom out/make things smaller/see more?